src/appkit.cs

```diff
@@ -26704,15 +26712,15 @@ interface NSButtonTouchBarItem
 
 		[Static]
 		[Export ("buttonTouchBarItemWithIdentifier:title:target:action:")]
-		NSButtonTouchBarItem ButtonTouchBarItemWithIdentifier (string identifier, string title, [NullAllowed] NSObject target, [NullAllowed] Selector action);
+		NSButtonTouchBarItem GetButtonTouchBarItem (string identifier, string title, [NullAllowed] NSObject target, [NullAllowed] Selector action);
```

Comment:
`Get` is a verb - but it sounds like you're not _getting_ something that exists but that you're _creating_ a new item

Reply:
Aah. Right! thanks!